src/main/java/sirius/biz/analytics/explorer/DataExplorerController.java

```diff
@@ -133,22 +162,61 @@ public Future chartApi(WebContext webContext, JSONStructuredOutput output) throw
-                                        comparisonPeriod,
-                                        output);
-            } catch (Exception e) {
-                throw Exceptions.handle(Log.APPLICATION, e);
+        List<Dataset> timeSeriesData = webContext.getParameters(PARAM_IDENTIFIER)
+                                                 .stream()
+                                                 .flatMap(identifier -> exportTimeSeriesAsDatasets(identifier,
+                                                                                                   timeSeries,
+                                                                                                   comparisonPeriod))
+                                                 .toList();
+
+        try (ExportXLSX export = new ExportXLSX(() -> webContext.respondWith()
+                                                                .download("data-explorer.xlsx")
+                                                                .outputStream(HttpResponseStatus.OK, null))) {
+            export.addListRow(Stream.concat(Stream.of(NLS.get("DataExplorerController.dateColumn")), timeSeriesData.stream().map(Dataset::getLabel))
+                                    .toList());
+            List<String> dates = timeSeries.startDates().map(date -> timeSeries.getGranularity().format(date)).toList();
+            for (int index = 0; index < dates.size(); index++) {
+                final int rowIndex = index;
+                export.addListRow(Stream.concat(Stream.of(dates.get(rowIndex)),
+                                                timeSeriesData.stream()
+                                                              .map(dataset -> dataset.getValues().get(rowIndex)))
+                                        .toList());
             }
-        });
+        }
+    }
+
+    private Stream<Dataset> exportTimeSeriesAsDatasets(String identifier,
+                                                       TimeSeries timeSeries,
+                                                       ComparisonPeriod comparisonPeriod) {
+
+        try {
+            Tuple<ChartFactory<Object>, Object> providerAndObject = resolveProviderAndObject(identifier);
+            return providerAndObject.getFirst()
+                                    .computeExportableTimeSeries(providerAndObject.getSecond(),
+                                                                 timeSeries,
+                                                                 comparisonPeriod)
+                                    .stream()
+                                    .map(timeSeriesData -> timeSeriesData.toDataset(timeSeries));
+        } catch (Exception e) {
+            Exceptions.handle()
+                      .to(Log.BACKGROUND)
+                      .error(e)
+                      .withSystemErrorMessage("DataExplorerController: Failed to compute time series for %s: %s (%s)",
```

Comment:
**Question:** No internationalization required?

Reply:
nope, this is a system-error message and "should not happen" anyway